assets/aircraft/atp.json

```diff
@@ -0,0 +1,37 @@
+{
+    "name": "British Aerospace ATP",
+    "icao": "ATP",
+    "engines": {
+        "number": 2,
+        "type": "S"
+    },
+    "weightClass": "M",
+    "category": {
+        "srs": 3,
+        "lahso": 6,
+        "recat": "D"
+    },
+    "ceiling": 25000,
+    "rate": {
+        "climb":      1000,
```

Comment:
As with the other aircraft in this PR I don't know which climb and descent values to use.